cfme/tests/services/test_service_rbac.py

```diff
@@ -113,6 +115,22 @@ def test_service_rbac_service_catalog(appliance, role_user_group, catalog, catal
         service_request.wait_for_request()
         assert service_request.is_succeeded()
 
+    @request.addfinalizer
+    def _finalize():
```

Comment:
Thanks for this cleanup :+1:
If user has access to delete request and VM; I think better to move this finalizer after `order`and we don't need to instantiate request again; order method will return it for you.